watertap/unit_models/tests/test_reverse_osmosis_0D.py

```diff
@@ -193,6 +195,25 @@ def test_option_pressure_change_calculated():
     assert isinstance(m.fs.unit.feed_side.N_Re, Var)
 
 
+@pytest.mark.unit
+def test_option_friction_factor_spiral_wound():
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(dynamic=False)
+    m.fs.properties = props.NaClParameterBlock()
+    m.fs.unit = ReverseOsmosis0D(
+        property_package=m.fs.properties,
+        has_pressure_change=True,
+        concentration_polarization_type=ConcentrationPolarizationType.calculated,
+        mass_transfer_coefficient=MassTransferCoefficient.calculated,
+        pressure_change_type=PressureChangeType.calculated,
+        friction_factor=FrictionFactor.spiral_wound,
+    )
+
+    assert m.fs.unit.config.friction_factor == FrictionFactor.spiral_wound
+    assert isinstance(m.fs.unit.feed_side.velocity, Var)
+    assert isinstance(m.fs.unit.feed_side.eq_friction_factor_spiral_wound, Constraint)
+
```

Comment:
I think you should do the same check for the other side of the membrane since this is OARO (counterflow operation).

Reply:
Additionally, it'd be nice to solve the model and check the solutions for friction factor values

Reply:
Thanks for the suggestions. I assume you meant to point to the oaro test file and not the RO0D test file (the variable & constraint in question do not exist on the permeate side of RO0D or RO1D)